src/drivers/dw/dma.c

```diff
@@ -557,9 +554,8 @@ static int dw_dma_set_config(struct dma_chan_data *channel,
 		if (dw_chan->lli)
 			rfree(dw_chan->lli);
 
-		dw_chan->lli = rballoc_align(0, SOF_MEM_CAPS_RAM | SOF_MEM_CAPS_DMA,
-					sizeof(struct dw_lli) * channel->desc_count,
-					PLATFORM_DCACHE_ALIGN);
+		dw_chan->lli = rballoc(0, SOF_MEM_CAPS_RAM | SOF_MEM_CAPS_DMA,
+				       sizeof(struct dw_lli) * channel->desc_count);
```

Comment:
again separate PR? This is unrelated to the allocation size, you're just changing how the LLI is handled.

Reply:
Looks like the rebase was on the wrong branch. These are already in @keyonjie other PRs

Reply:
> again separate PR? This is unrelated to the allocation size, you're just changing how the LLI is handled.

Ah, yes, let me put them to PR #4664

Reply:
alignment?

Reply:
> alignment?

the rballoc() internal will use default PLATFORM_DCACHE_ALIGN alignment, so no need to specify it explicitly.